lib/src/parse/css.dart

```diff
@@ -32,7 +32,9 @@ class CssParser extends ScssParser {
 
   CssParser(super.contents, {super.url, super.logger});
 
-  void silentComment() {
+  bool silentComment() {
```

Comment:
We don't, but we _do_ want to throw an explicit error when we encounter them. This is particularly true because historically LibSass parsed `.css` files as SCSS and some frameworks took advantage of this, so there are _definitely_ real world cases where the user needs a better error than whatever parsing failure they'd get in plain CSS.